src/atoms/UserImage/UserImage.tsx

```diff
@@ -0,0 +1,28 @@
+import React, { useMemo } from 'react';
+
+import { ICompanyDataProps, IStudentDataProps } from 'context';
+import { StyledUserImage, UserInitials } from './styled';
+import { getUserInitials } from './utils/getUserInitials';
+
+export interface IUserImageProps {
+  user: IStudentDataProps | ICompanyDataProps,
+  onImageClick: () => void,
+}
+
+export const UserImage: React.FC<IUserImageProps> = ({ user, onImageClick }) => {
+  const userInitials = useMemo(() => getUserInitials(user), [user]);
+
+  return (
+    <>
+      { 'img' in user ?
```

Comment:
`'img' in user`
Why not `Boolean(user.img)`?
Or just `user.img ? true : false`
user.img will be converted to boolean value in ternary operator.

Reply:
By the way, I think UserImage component should take undefined too. And show default picture if there is undefined.